test/components/util/util.go

```diff
@@ -458,3 +459,29 @@ func IsPlatformSupported(t *testing.T, platforms []Platform) bool {
 
 	return false
 }
+
```

Comment:
This also needs to be used by the testing code we have right now to be effective.